src/client/pydaos/raw/daos_api.py

```diff
@@ -1394,6 +1394,31 @@ def __init__(self):
         self.chunk_size = ctypes.c_uint64(0)
 
 
+class DaosInputParams(object):
+    """ This is a helper python method
+    which can be used to pack input
+    parameters for create methods
+    (eg: container or pool (future)).
+    """
+    def __init__(self):
+        super(DaosInputParams, self).__init__()
+        # Get the input params for setting
+        # container properties for
+        # create method.
+        self.co_prop = DaosContProperties()
+
+    def get_con_create_params(self):
+        """ Get the container create params.
+        This is method is used to pack
```

Comment:
Typo. "This is method is"

Reply:
Done. Since Functional hardware test failed for no reason, I thought I will fix this, merge with master and re-run the tests once again.